### PR TITLE
Removed the bithound badges, restored the CC badges

### PR DESCRIPTION
**Pull request by chriskacerguis**

### CODE REVIEW

**Comment by Fryguy** [comment]
Are you putting NSP back?  If so, you might want to just `give revert 259f243`

**Comment by Fryguy** [comment]
Also, if you're killing bitHound, you'll need to kill the services and hooks.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,10 @@
 # ManageIQ Service UI
 
 [![Build Status](https://travis-ci.org/ManageIQ/manageiq-ui-service.svg?branch=master)](https://travis-ci.org/ManageIQ/manageiq-ui-service)
-[![Test Coverage](https://codeclimate.com/github/ManageIQ/manageiq-ui-service/badges/coverage.svg)](https://codeclimate.com/github/ManageIQ/manageiq-ui-service/coverage)
+[![Test Coverage](https://codeclimate.com/github/ManageIQ/manageiq-ui-service/badges/coverage.svg)](https://codeclimate.com/github/ManageIQ/manageiq-ui-service/coverage) [![Code Climate](https://codeclimate.com/github/ManageIQ/manageiq-ui-service/badges/gpa.svg)](https://codeclimate.com/github/ManageIQ/manageiq-ui-service)
```
Comment (Fryguy) [inline README.md]: These badges should be on two separate lines

**Comment by chriskacerguis** [comment]
@Fryguy was just going to add that in CC

**Comment by chriskacerguis** [comment]
hooks / integrations removed.

**Comment by Fryguy** [comment]
@chriskacerguis Since all this does it change a README, squash your commits, put `[skip ci]` in the commit message body, and then it can ignore travis.  Not sure what Travis' problem in this PR is.

**Comment by AllenBW** [comment]
Its gonna be one of those random travis fail days 
`The command "eval travis_start_sauce_connect " failed. Retrying, 2 of 3.`

**Comment by Fryguy** [comment]
https://github.com/ManageIQ/manageiq-ui-service/pull/747#issuecomment-300182047 ^ ?

**Comment by chriskacerguis** [comment]
@Fryguy NSP is one of the CodeClimate engines, so was just going to add that as an engine.

**Pull request closed without merging**